File: CS/WinWebSolution.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            using (UnitOfWork uow = new UnitOfWork(Session.DataLayer)) {
                ContactType1 ct1 = new ContactType1(uow);
                ct1.FirstName = "Contact 1 (Type 1)";
                ct1.Active = true;

                ContactType2 ct2 = new ContactType2(uow);
                ct2.FirstName = "Contact 2 (Type 2)";

                ContactType2 ct3 = new ContactType2(uow);
                ct2.Active = true;
                ct3.FirstName = "Contact 3 (Type 2)";
                ct3.Active = false;

                DocumentType1 dt1 = new DocumentType1(uow);
                dt1.Name = "Document 1 (Type 1)";
                dt1.CreatedBy = ct1;

                DocumentType2 dt2 = new DocumentType2(uow);
                dt2.Name = "Document 2 (Type 2)";
                dt2.Author = "Author 1 (Type 2)";
                dt2.CreatedBy = ct2;
                
                uow.CommitChanges();
            }
        }
    }
}

File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
			MyBase.New(session, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Using uow As New UnitOfWork(Session.DataLayer)
				Dim ct1 As New ContactType1(uow)
				ct1.FirstName = "Contact 1 (Type 1)"
				ct1.Active = True

				Dim ct2 As New ContactType2(uow)
				ct2.FirstName = "Contact 2 (Type 2)"

				Dim ct3 As New ContactType2(uow)
				ct2.Active = True
				ct3.FirstName = "Contact 3 (Type 2)"
				ct3.Active = False

				Dim dt1 As New DocumentType1(uow)
				dt1.Name = "Document 1 (Type 1)"
				dt1.CreatedBy = ct1

				Dim dt2 As New DocumentType2(uow)
				dt2.Name = "Document 2 (Type 2)"
				dt2.Author = "Author 1 (Type 2)"
				dt2.CreatedBy = ct2

				uow.CommitChanges()
			End Using
		End Sub
	End Class
End Namespace
